bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -136,25 +136,21 @@ rules:
   - privileged
 - apiGroups: ["k8s.ovn.org"]
   resources:
-  - adminpolicybasedexternalroutes
-  - adminpolicybasedexternalroutes/status
-  - egressfirewalls
-  - egressfirewalls/status
-  - egressqoses
-  verbs:
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups: ["k8s.ovn.org"]
-  resources:
+    - adminpolicybasedexternalroutes
+    - egressfirewalls
     - egressips
+    - egressqoses
     - egressservices
   verbs:
     - get
     - list
     - watch
+- apiGroups: ["k8s.ovn.org"]
+  resources:
+  - adminpolicybasedexternalroutes/status
+  - egressfirewalls/status
+  verbs:
+  - update
```

Comment:
I think it does patch instead of update, but we usually include them together in case the method changes

Reply:
yes @npinaeva , good catch. Both APBExternalRoute, EgressFirewall finally does Patch call (via ApplyStatus) for updating its status, changed it and hope it would work. 
Also updated the control-plane permissions.

Reply:
oh I think I missed that this PR also changes egressQoS on purpose. I can only lgtm for egress firewall and APBroute